backend/src/models/User.ts

```diff
@@ -18,7 +19,34 @@ const UserSchema = new Schema<User, UserModel>({
     type: String,
```

Comment:
this should just be multiple booleans like user.student=true or false like you had earlier

Reply:
![image](https://user-images.githubusercontent.com/55229818/104949843-9af5e880-598d-11eb-9ef3-e451e9939068.png)
don't think we need booleans can just use instanceof

Reply:
https://mongoosejs.com/docs/discriminators.html